src/gt4sd/frameworks/enzeptional/processing.py

```diff
@@ -232,3 +324,39 @@ def reconstruct_sequence_with_mutation_range(
         mutated_sequence_offset = end + 1
     mutated_sequence += sequence[end + 1 :]
     return mutated_sequence
+
+
+def selection(scores, k=20):
+    """Selecte the top K mutated sequences based on the score
```

Comment:
check the docstrings there are typos, K should be lower case and we also have to report the default.

Reply:
typing missing